lib/rules/no-property-shorthands.js

```diff
@@ -13,18 +13,64 @@ module.exports = {
             url:
                 "http://mysticatea.github.io/eslint-plugin-es/rules/no-property-shorthands.html",
         },
-        fixable: null,
+        fixable: "code",
         schema: [],
         messages: {
             forbidden: "ES2015 property shorthands are forbidden.",
         },
     },
     create(context) {
+        const sourceCode = context.getSourceCode()
+
+        /**
+         * Fixes a FunctionExpression node by making it into a longform property.
+         * @param {SourceCodeFixer} fixer The fixer object
+         * @param {ASTNode} node A `Property` node that has a `FunctionExpression` as its value
+         * @returns {object} A fix for this node
+         */
+        function makeFunctionLongform(fixer, node) {
+            const firstKeyToken = node.computed
+                ? sourceCode.getTokens(node).find(token => token.value === "[")
```

Comment:
I guess that this line can be more efficient a bit:

```suggestion
                ? sourceCode.getTokenBefore(node.key, isOpeningBracketToken)
```

※ [isOpeningBracketToken](https://mysticatea.github.io/eslint-utils/api/token-utils.html#isopeningbrackettoken-isnotopeningbrackettoken)

Reply:
Thank you for suggestion!
I changed it. Please check again.